### PR TITLE
Docs: Update references of `grafana-cli` to `grafana cli` and `grafana-server` to `grafana server`.

### PR DESCRIPTION
**Pull request by kminehart**
According to https://github.com/grafana/grafana/pull/66976, these two commands are deprecated as of Grafana 10, so we should update the documentation to reflect how we would prefer users invoke the Grafana CLI and server.

There are still references to `grafana-server` in the context of the systemd service; this is not changing so the systemd service name 'grafana-server' should remain.

### CODE REVIEW

File: docs/sources/cli.md [hunk 1]
```diff
@@ -18,12 +18,12 @@ Grafana CLI is a small executable that is bundled with Grafana server. It can be
...
 
 ```
-grafana-cli -h
+grafana cli -h
 ```
 
 ## Invoking Grafana CLI
 
-To invoke Grafana CLI, add the path to the grafana binaries in your `PATH` environment variable. Alternately, if your current directory is the `bin` directory, use `./grafana-cli`. Otherwise, you can specify full path to the CLI. For example, on Linux `/usr/share/grafana/bin/grafana-cli` and on Windows `C:\Program Files\GrafanaLabs\grafana\bin\grafana-cli.exe`.
+To invoke Grafana CLI, add the path to the grafana binaries in your `PATH` environment variable. Alternately, if your current directory is the `bin` directory, use `./grafana cli`. Otherwise, you can specify full path to the CLI. For example, on Linux `/usr/share/grafana/bin/grafana cli` and on Windows `C:\Program Files\GrafanaLabs\grafana\bin\grafana cli.exe`.
```
Comment (tolzhabayev) [inline docs/sources/cli.md:26]: `and on Windows `C:\Program Files\GrafanaLabs\grafana\bin\grafana cli.exe`.

this seems broken - empty space before exe?

Comment (kminehart) [reply docs/sources/cli.md:26]: ah good catch.

Comment (kminehart) [reply docs/sources/cli.md]: fixed

**Review by chri2547 — COMMENTED** [review]
Soft approving this PR so as to get feedback from @tolzhabayev

**Review by tolzhabayev — APPROVED** [review]

**Pull request merged**